tests/examples/Fervo_Project_Cape.txt

```diff
@@ -4,7 +4,7 @@
 Reservoir Model, 1
 Reservoir Volume Option, 1
 Reservoir Density, 2800
-Reservoir Depth, 8500 feet, -- https://pangea.stanford.edu/ERE/db/GeoConf/papers/SGW/2024/Fercho.pdf
```

Comment:
This would be a backwards-incompatible change - can we preserve `feet` as a valid unit?

Reply:
If I understand [the Units classes](https://github.com/NREL/GEOPHIRES-X/blob/main/src/geophires_x/Units.py) correctly, a given unit can have only one value (e.g., in `LengthUnit`, the value for `FEET` is "ft").  The only way I see to allow multiple values for a given unit would be to add a special case into [LookupUnits](https://github.com/NREL/GEOPHIRES-X/blob/main/src/geophires_x/Parameter.py#L747), but this strikes me as a hack.  Is there another (or a better) way?

Reply:
We might be able to add an entry in https://github.com/NREL/GEOPHIRES-X/blob/main/src/geophires_x/GEOPHIRES3_newunits.txt